tests/core/helpers.py

```diff
@@ -132,13 +137,17 @@ def __init__(self, parent, trained_model, save_path, save_with_id):
             super().__init__(
                 trainer_name=parent.instance_name,
                 training_id=str(uuid.uuid4()),
-                save_path=save_path,
                 save_with_id=save_with_id,
             )
             self._parent = parent
             self._trained_model = trained_model
             self._canceled = False
             self._completed = False
+            self._save_path = save_path
+
+        @property
+        def save_path(self):
```

Comment:
I think this change makes sense because `save_path` is really only relevant to the local model trainer but want to double check.